src/main/java/org/opensearch/ad/ml/EntityColdStarter.java

```diff
@@ -597,7 +597,7 @@ private int calculateColdStartDataSize(List<double[][]> coldStartData) {
     private Pair<Integer, Integer> selectRangeParam(AnomalyDetector detector) {
```

Comment:
Not a part of this PR: but move this to util? Pair<X,Y> should be useful at multiple places ....

Reply:
Pair is from apache commons: org.apache.commons.lang3.tuple.Pair. We can use it elsewhere.

Reply:
My mistake, makes sense.